build.sbt

```diff
@@ -25,12 +25,14 @@ lazy val scalaPyTensorFlow = project
   .in(file("."))
```

Comment:
For the record: `sbt publishM2` results in the following artifacts getting published:

![image](https://user-images.githubusercontent.com/3383210/89796169-92210100-db29-11ea-89fd-967d82dd293a.png)

Reply:
...which is perfectly okay AFAICS ;D